### PR TITLE
Make country field a drop down and print country names

### LINKED ISSUE #41: The country field should be a nice select box of ISO3166 countries
The database field is ready to store the two-digit code, and we have the right ruby gem available to turn this into a country with a nice name, but we need to link the two up.

For #23 and #8

### PR DESCRIPTION
**Pull request by stevenday**
Closes #41

### CODE REVIEW

File: lib/tasks/load_msf_spreadsheet.rake [hunk 1]
```diff
@@ -36,6 +36,17 @@ task :load_msf_spreadsheet, [:csv_file] => [:environment] do |_t, args|
       date = Date.strptime(row[:concept_paper_date], "%d/%m/%Y")
     end
 
+    country_code = nil
+    unless row[:study_location].blank?
+      study_location = row[:study_location]
+      # The countries gem is very particular about some names
+      if study_location == "Democratic Republic of the Congo"
+        study_location = "Congo, The Democratic Republic Of The"
+      end
```
Comment (davea) [inline lib/tasks/load_msf_spreadsheet.rake]: Also Burma/Myanmar.

**Comment by davea** [comment]
There are some studies that span multiple countries (e.g. 'CAR, South Sudan and DRC'). Does this model consider each study to have a single 'main' country, or should it allow multiple countries per study?

File: lib/tasks/load_msf_spreadsheet.rake (hunk 1 above)
Comment (stevenday) [inline lib/tasks/load_msf_spreadsheet.rake]: Ooh, good spot! Thanks, I've added that too

**Comment by stevenday** [comment]
With regards to multiple countries, I'll ask MSF what they think, but I have a vague recollection of us discussing this before and saying it wasn't an issue (certainly, their schema didn't allow for it). That can always be a new issue though.

**Pull request closed without merging**